doc/source/whatsnew/v0.17.1.txt

```diff
@@ -64,6 +64,7 @@ Performance Improvements
 - Improved performance of ``rolling_median`` (:issue:`11450`)
 
 - Improved performance to ``to_excel`` (:issue:`11352`)
+- Improved performance of ``andrews_curves``
```

Comment:
add the issue number here (use this pr number as their is not an associated issue)